main.js

```diff
@@ -4,19 +4,194 @@ const navClose = document.getElementById('close-menu');
+        <ul class="skill">
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag0.png"
+                        alt="Rubi legend, as reference of the back end language used on the project"></a>
+            </li>
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag1.png"
+                        alt="Bootstrap legend, as reference of the tool used for styling the page"></a></li>
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag2.png"
+                        alt="HTML legend, as reference of the coding used on the front end"></a></li>
+        </ul>
+        <button data-target="${projectInformation[counter].id}" type="submit" class="projectbutton">See Project</button>
+    </div>`;
+  }
+
+  function hideModal() {
+    popUp.innerHTML = '';
+    popUp.classList.add('hidden');
+  }
+
+  function showModal(button) {
+    const pId = button.target.getAttribute('data-target');
+    const project = projectInformation.filter((p) => parseInt(p.id, 10) === parseInt(pId, 10));
+
+    popUp.innerHTML = `<div class="projectdetails">
+            <div class="projectdetails-title">
+                <h3 id="popUpTitle" class="pop-title">${project[0].name}</h3>
+                <p id="popUpClose"><img id="popUpCloseImage" src="./images/icn-close.svg" alt="Exit Icon"></p>
+            </div>
+            <div>
+                <ul class="projectdetails-skills">
+                    <li><a href="#"><img src="./images/popup/Tag0.svg" alt="HTLM icon that leads to the html code"></a>
+                    </li>
+                    <li><a href="#"><img src="./images/popup/Tag1.svg"
+                                alt="Bootstrap icon that leads to the bootstrap format used on the project"></a></li>
+                    <li><a href="#"><img src="./images/popup/Tag2.svg"
+                                alt="Ruby on Rails icon that leads to the code on this lenguage that was used on the project"></a>
+                    </li>
+                </ul>
+            </div>
+            <div class="projectdetails-content">
+                <div class="projectdetails-image">
+                    <img class="pop-img" src="${project[0].image}"
+                        alt="A half scrren image that shows the first impresion of the results after the coding implementation">
+                </div>
+                <div class="projectdetails-body">
+                    <p class="pop-paragraph">${project[0].description}</p>
+                    <p class="anchorarea">  
+                      <a class="anchorbtn" id="projectDetailButtonLive" href = ${project[0].liveVersionLink} target="_blank">See Live <img src="./images/popup/Icon-see-live.svg" alt=""></a>
+                      <a class="anchorbtn" id="projectDetailButtonSource" href =${project[0].sourceLink} target="_blank">See Source <img src="./images/popup/Vector.svg" alt=""></a>
+                    </p>
```

Comment:
[OPTIONAL] Kindly align the buttons to the bottom so your design matches the Figma design. 
- Your design:
<img width="884" alt="image" src="https://user-images.githubusercontent.com/77190203/156662399-f16fd197-49ca-444a-b1dd-7bd181cb89f9.png">

- Figma design:
<img width="324" alt="image" src="https://user-images.githubusercontent.com/77190203/156662573-2e19ac2a-adcd-4b4b-90c1-ee0850ad7def.png">